_pytest/test_components.py

```diff
@@ -85,7 +85,7 @@ def test_all_arguments_can_be_satisfied_during_parse(component_class, default_co
 
     # All available context arguments that will ever be generated during parse
     component = component_builder.create_component(component_class.name, default_config)
-    context_arguments = {"text": None}
+    context_arguments = {"text": None, "ref_time": ""}
```

Comment:
Also, I'd like to rename this to something that expresses that this is the time of the message. Do you have a suggestion here, maybe just `time`?